src/index.html

```diff
@@ -11,7 +11,173 @@
+            Password:
+            <input
+              class="form-input"
+              type="password"
+              id="password"
+              name="password">
+          </div>
+        </fieldset class="fieldset-block">
+      </div>
+      <div class="about">
+        <fieldset class="fieldset-block">
+          <legend>An interesting fact about you!</legend>
+          <div class="form-field">
+            Do you love cats?
+            <input
+              class="form-input"
+              type="radio"
+              id="loveCats"
+              name="loveCatsYes"
+              value="y">
+            Yes
+            <input
+              class="form-input"
+              type="radio"
+              id="loveCatsNo"
+              name="loveCats"
+              value="n">
+            No
+          </div>
+          <div class="form-field">
+            What is your favorite color?
+            <input
+              class="form-input"
+              type="color"
+              id="color"
+              name="color">
+          </div>
+          <div class="form-field">
+            What time do you go to bed?
+            <input
+              class="form-input"
+              type="time"
+              id="sleepTime"
+              name="sleepTime"
+              step="1">
+          </div>
+          <div class="form-input">
+            <label for="cars"> What are your favorite brand of cars?
+              <select
+                name="cars"
+                id="cars"
+                multiple>
+                <option value="">BMW</option>
+                <option value="">Audi</option>
+                <option value="">Lada</option>
+                <option value=""></option>
+              </select>
+            </label>
+          </div>
+          <div class="form-field">
+            How do you rate our work?
+            <input
+              class="form-input"
+              type="range"
+              name="rateUs"
+              id="rateUs"
+              value="0">
+          </div>
+        </fieldset class="fieldset-block">
+      </div>
+      <div class="additionalInfo">
+        <fieldset class="fieldset-block">
+          <legend>Additional info:</legend>
+
+          <div class="form-input">
+            Comments:
+            <textarea
+              name="comments"
+              id="comments"
+              cols="20"
+              rows="2">
+            </textarea>
+          </div>
+
+          <div class="recomendUs">
+            <label for="recommendUs">Would you recommend us?</label>
+            <select name="recommendUs" id="recommendUs">
+              <option value="yes">yes</option>
+              <option value="no">no</option>
+            </select>
+          </div>
+        </fieldset class="fieldset-block">
+      </div>
+
+      <button type="submit">
+        Submit
+      </button>
+    </form>
     <script type="text/javascript" src="./main.js"></script>
   </body>
-</html>
+</html>
```

Comment:
```suggestion
</html>

```
Додай пустий рядок наприкінці коду